.github/policies/resourceManagement.yml

```diff
@@ -0,0 +1,34 @@
+name: GitOps.PullRequestIssueManagement
+description: GitOps.PullRequestIssueManagement primitive
+owner: null
```

Comment:
What's the effect of not having an owner?

Reply:
No effect as we don't really use that property. The property can just be removed though too.